### ISSUE TITLE
Issue with ember 2.10.0

### ISSUE THREAD

**Issue by deanmraz** [rynam0/ember-swagger-ui#38]
# Issue
Doesn't work with ember 2.10.0 

# Error 

```
Uncaught TypeError: Cannot create property 'default' on boolean 'true'
    at new module.exports (vendor.js:84168)
    at vendor.js:81201
    at arrayEach (vendor.js:91925)
    at Object.forEach (vendor.js:93258)
    at vendor.js:81180
    at vendor.js:93140
    at baseForOwn (vendor.js:92345)
    at vendor.js:93105
    at Object.forEach (vendor.js:93259)
    at module.exports.SwaggerClient.buildFromSpec (vendor.js:81174)
```

# Demo Project 
https://github.com/deanmraz/ember-swagger-demo

### Implementation
https://github.com/deanmraz/ember-swagger-demo/blob/master/app/templates/index.hbs

@rynam0 I was hoping you can point me in the right direction to fix this. Let me know if you have any suggestions.

**Comment by rynam0** [comment]
Thanks for reporting this. I'll take a look at this tomorrow and hopefully I'll have a doc for you shortly after.

**Comment by rynam0** [comment]
Thanks to the demo project you created I was able to find a quick workaround that I hope will work for you.
I'm not entirely clear on why yet but it seems that if you disable prototype extensions for Arrays, this error goes away.
You can achieve this by updating `config/environment.js` to look like the following:

    EXTEND_PROTOTYPES: {
        // Prevent Ember Data from overriding Date.parse.
        Date: false,
        Array: false
    }

I'm going to close this for now but please feel free to reopen if this doesn't work for you or if you feel like we need to look into getting something fixed upstream (swagger).  Thanks again!

**Comment by deanmraz** [comment]
@rynam0 thanks for the help! I was also able to fix this by upgrading swagger ui to 2.2.10.

**Comment by rynam0** [comment]
Fantastic. Thanks for the update!
